docs/terraformoptions.adoc

```diff
@@ -141,11 +141,21 @@ Configuration Terraform Options:
 |XXX.XXX.XXX.XXX/YY
 |ANYWHERE
 
+|bastion_image_id
+|Custom image id for the bastion host
+|image id or NONE. If the value is set to NONE, an Oracle Platform image will be used instead. Set use_autonomous to _false_ if you want to use your own image.
+|NONE
+
 |bastion_shape
 |The shape of bastion instance.
 |
 |VM.Standard.E2.1
 
+|bastion_upgrade
+|Whether to upgrade the bastion host after provisioning. It's useful to set this to false during development so the bastion is provisioned faster.
```

Comment:
If `bastion_upgrade` is `true`, then the bastion is upgraded to what after it is provisioned -- latest patches for the selected OS version, latest versions of selected packages, or something else? Pls clarify.

Also, if the upgrade is a post-provisioning activity, then how does setting this to `false` improve the provisioning time? Did you mean that bastion is unavailable during the upgrade process?

Reply:
This is for package upgrade only. Will clarify in the docs.

Reply:
The package upgrade is done by cloud-init, so the bastion is only available after cloud-init has finished.